basedtyping/__init__.py

```diff
@@ -28,6 +31,10 @@
     # for isinstance checks
     Function = Callable
 
+
+Never = NoReturn
+"""a type that can never exist. this is the narrowest possible type"""
```

Comment:
Would be good to mention it's a type form not a `type`

Reply:
cant now that its a re-export, had to delete the doc